aion/web/channels/room_channel.ex

```diff
@@ -101,8 +105,13 @@ defmodule Aion.RoomChannel do
   end
 
   defp send_current_question(room_id, socket) do
+    # NOTE: This function is called every time a user joins room / the question changes
```

Comment:
`@doc` ?

Reply:
I've added the comment concerning before:
![image](https://user-images.githubusercontent.com/15965147/31276175-d03589f6-aa9a-11e7-9580-57e2b741a967.png)

I thought it may be better to have
```
@doc

@spec
def

@spec
def
```

rather than

```
@doc
@spec
def

@spec
def
```
because in the second approach I was having an impression that the `@doc` is connected to only  the first clause. In the first approach, though, you may get the impression that it's a random doc. Should I remove the line between`@doc` and `@spec`?